cityofaustin/atd-moped
# Fix status badge Active phase bug

# Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9848

I tried to repeat the steps Meredith took and was unable to reproduce an "Active" badge by following the steps in the activity log for the project mentioned in the issue. However, there is a bug in production where if you change the status name of a non-current status, the badge reverts to "Active" instead of keeping the name of the current status. 

I moved around some of the logic so the badge name doesnt change unless the current status is being edited. 


## Testing
**URL to test:** 

https://deploy-preview-748--atd-moped-main.netlify.app/moped/projects/1866?tab=timeline

**Steps to test:**

1. Create a new project
2. navigate to Timeline Tab, Potential is the current status. Badge should say Potential. 
3. Toggle the Potential status current switch to false. Badge should be green and Active
4. Add a few more statuses. Mark one as Current. That status name should be on the badge. 
5. Change the name of an existing (non-current) status. Badge name should not change.
6. Change the name of the existing current status. Badge name should change to match the new name. 

---
#### Ship list
